Utilities/ocpasswordgen/ocpasswordgen.c

```diff
@@ -21,13 +58,34 @@ ENTRY_POINT (
   UINT8   Index;
   UINT8   PasswordHash[SHA512_DIGEST_SIZE];
 
-  DEBUG ((DEBUG_ERROR, "Please enter your password: "));
+  DEBUG ((DEBUG_ERROR, "Please enter your password: \n"));
 
   for (PasswordLen = 0; PasswordLen < OC_PASSWORD_MAX_LEN; ++PasswordLen) {
-    Char = getchar ();
-    if ((Char == EOF) || (Char == '\n')) {
+    fflush(stdin);
+    Char = getch ();
+    // Handle interrupt signal for conio
+    if (Char == CHAR_END_OF_TEXT) {
+      exit(EXIT_SUCCESS);
+    }
+    if (Char == EOF || Char == CHAR_LINEFEED) {
       break;
     }
+    if (Char == CHAR_CARRIAGE_RETURN) {
```

Comment:
else if? and same below.

Reply:
No, since break does not return control.